src/Xamarin.Android.Build.Tasks/Tasks/GenerateJniRemappingNativeCode.cs

```diff
@@ -131,7 +131,7 @@ void ReadXml (XmlReader reader, List<JniRemappingTypeReplacement> typeReplacemen
 					}
 
 					if (!Boolean.TryParse (targetIsStatic, out bool isStatic)) {
-						Log.LogError ($"Attribute 'target-method-instance-to-static' in element '{reader.LocalName}' value '{targetIsStatic}' cannot be parsed as boolean; {RemappingXmlFilePath.ItemSpec} line {GetCurrentLineNumber ()}");
+						Log.LogError ($"Attribute 'target-method-instance-to-static' in element '{reader.LocalName}' value '{targetIsStatic}' cannot be parsed as boolean; {RemappingXmlFilePath!.ItemSpec} line {GetCurrentLineNumber ()}");
```

Comment:
Instead of using `!` here, can we pass in a non-nullable `string remappingXmlFilePath`.